jdbc/src/main/java/io/kestra/jdbc/repository/AbstractJdbcMetricRepository.java

```diff
@@ -317,7 +336,44 @@ public Function<String, String> sortMapping() throws IllegalArgumentException {
     }
 
     @Override
-    public ArrayListTotal<Map<String, Object>> fetchData(String tenantId, DataFilter<Metrics.Fields, ? extends ColumnDescriptor<Metrics.Fields>> filter, ZonedDateTime startDate, ZonedDateTime endDate, Pageable pageable) throws IOException {
```

Comment:
It seems the 3 fetchData methods are the same on each DB.
Maybe you can move them to the parent class?